set.mm

```diff
@@ -35244,8 +35266,18 @@ practical reasons (to avoid having to prove sethood of ` A ` in every use
   ${
     $d x y A $.  $d y ph $.
     $( An equivalence for class substitution.  (Contributed by NM,
-       23-Aug-1993.)  (Revised by Mario Carneiro, 12-Oct-2016.) $)
+       23-Aug-1993.)  (Revised by Mario Carneiro, 12-Oct-2016.)  (Proof
+       shortened by SN, 2-Sep-2024.) $)
     sbc5 $p |- ( [. A / x ]. ph <-> E. x ( x = A /\ ph ) ) $=
+      ( wsbc cab wcel cv wceq wa wex df-sbc clelab bitri ) ABCDCABEFBGCHAIBJABC
+      KABCLM $.
+
+    $( Alternate proof of ~ sbc5 .  This proof is equivalent to the proof of
+       ~ clelab but comes after ~ vtoclbg and ~ isset , which aids
+       understanding of the proof of ~ clelab .  (Contributed by NM,
```

Comment:
If this proof does nothing more than "inlining" the proof of clelab in the proof of sbc5, the it can probably be obsoleted as an *OLD ?

Reply:
the proof of clelab basically expands vtoclbg (et al vtocl*) and replaces various `A e. _V` conveniences with `E. y y e. A`, and so is much more complicated. I think sbc5 should be kept for clarity